code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -142,6 +142,7 @@
 //ATTACK HAND IGNORING PARENT RETURN VALUE
 /mob/living/carbon/attack_hand(mob/living/carbon/human/user)
 
+	SEND_SIGNAL(src, COMSIG_ATOM_ATTACK_HAND, user)
```

Comment:
this is kind of shit but I don't think there's a simple way around it while this proc wont call parent